src/main/resources/db/migration/v0.30.0.1__linkml_v2.0.0_update.sql

```diff
@@ -0,0 +1,2574 @@
+ALTER TABLE stageterm DROP CONSTRAINT fkas58x03rc132q00y838dv1gsb;
+ALTER TABLE stageterm_aud DROP CONSTRAINT fkbe4dl5s3i7ga7hqryddog2g0f;
+ALTER TABLE synonym_aud DROP CONSTRAINT fk8y4re95uhgku1km6nsauced0b;
+ALTER TABLE uberonterm DROP CONSTRAINT uberonterm_curie_fk;
+ALTER TABLE uberonterm_aud DROP CONSTRAINT uberonterm_aud_curie_rev_fk;
+ALTER TABLE variant DROP CONSTRAINT variant_curie_fk;
+ALTER TABLE variant DROP CONSTRAINT variant_sourcegeneralconsequence_curie_fk;
+ALTER TABLE variant DROP CONSTRAINT variant_varianttype_curie_fk;
+DROP INDEX variant_sourcegeneralconsequence_index;
+DROP INDEX variant_varianttype_index;
+ALTER TABLE variant_aud DROP CONSTRAINT variant_aud_curie_rev_fk;
+ALTER TABLE variant_note DROP CONSTRAINT variant_note_variant_curie_fk;
+DROP INDEX variant_note_variant_curie_index;
+ALTER INDEX variant_note_relatednotes_id_index RENAME TO variant_note_relatednotes_index;
+ALTER TABLE vocabulary_aud DROP CONSTRAINT fk2pe60ji8kxipj6qf3cd6hjda5;
+ALTER TABLE vocabularyterm_aud DROP CONSTRAINT fkfg8df1h10nehkt3pw4cebbde8;
+ALTER TABLE vocabularytermset_aud DROP CONSTRAINT fkkqfblby0u8l1h2i3fm7h4ynlh;
+ALTER TABLE vtterm DROP CONSTRAINT vtterm_curie_fk;
+ALTER TABLE vtterm_aud DROP CONSTRAINT vtterm_aud_curie_rev_fk;
+ALTER TABLE wbbtterm DROP CONSTRAINT fkqnxqrnadcxojeti2ienobdqh0;
+ALTER TABLE wbbtterm_aud DROP CONSTRAINT fkhu85m34h8hf95s453u3m4ed8y;
+ALTER TABLE wblsterm DROP CONSTRAINT fk4k9uvngmwim574bm5mu5q4203;
+ALTER TABLE wblsterm_aud DROP CONSTRAINT fkbug8ndvvjf8e7e3rpcih06j63;
+ALTER TABLE wbphenotypeterm DROP CONSTRAINT fkhonuwqm6g91le1xigmik3yrwl;
+ALTER TABLE wbphenotypeterm_aud DROP CONSTRAINT fk48fs5fxgn3sfvyyiqpgoquuma;
+ALTER TABLE xbaterm DROP CONSTRAINT fkcnuymf0pvlgwcfd918o4pgs5n;
+ALTER TABLE xbaterm_aud DROP CONSTRAINT fkha6obinkag86qlcpemxmgv4ly;
+ALTER TABLE xbedterm DROP CONSTRAINT fkc0dct519510hodhd9d7phfq6g;
+ALTER TABLE xbedterm_aud DROP CONSTRAINT fkshxj981p427yuk4qgtvcirp5k;
+ALTER TABLE xbsterm DROP CONSTRAINT fkn8tvxj5qdpjssxjl9jneqst92;
+ALTER TABLE xbsterm_aud DROP CONSTRAINT fkij9o8qar117chfev5ghkupl3o;
+ALTER TABLE xcoterm DROP CONSTRAINT fk35ywtb8qiadqbwsb706ebu81c;
+ALTER TABLE xcoterm_aud DROP CONSTRAINT fknlbuiyo3i6daerkmpim317bd3;
+ALTER TABLE xpoterm DROP CONSTRAINT fkqeoygtsa6sf94gw089ugocoph;
+ALTER TABLE xpoterm_aud DROP CONSTRAINT fkbqybirrf1obv6esgeaj8os211;
+ALTER TABLE xsmoterm DROP CONSTRAINT fk36v85u2ghsukqljkkh1jxmtlx;
+ALTER TABLE xsmoterm_aud DROP CONSTRAINT fkjdlhxyvw79i14932dqdb1bx8f;
+ALTER TABLE zecoterm DROP CONSTRAINT fkqd3f6hcopl67fwai6viq07t88;
+ALTER TABLE zecoterm_aud DROP CONSTRAINT fke5wuchgyjhb2orgvht50q2dah;
+ALTER TABLE zfaterm DROP CONSTRAINT fk572s3xiqi0y4gjblq8xjyk3f7;
+ALTER TABLE zfaterm_aud DROP CONSTRAINT fks66s1k4fon0to2kk7qfsm1xon;
+ALTER TABLE zfsterm DROP CONSTRAINT fknkrcnh3l23ol126v9w61nk079;
+ALTER TABLE zfsterm_aud DROP CONSTRAINT fk27dwwh5ekpa60ug4ystuwwn6w;
+
+ALTER TABLE affectedgenomicmodel DROP CONSTRAINT affectedgenomicmodel_pkey;
```

Comment:
Dropping the primary keys does not help with the later query of: `UPDATE affectedgenomicmodel t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'biologicalentity';` as t.curie now has no index.